tmux/PKGBUILD

```diff
@@ -3,6 +3,7 @@
 pkgname=tmux
 pkgver=2.6
 pkgrel=1
+epoch=1
```

Comment:
Git for Windows does not modify `tmux`. Therefore this change should be unnecessary.